examples/gameroom/daemon/src/rest_api/routes/gameroom.rs

```diff
@@ -22,7 +22,7 @@ use gameroom_database::{
 use openssl::hash::{hash, MessageDigest};
 use protobuf::Message;
 use splinter::admin::messages::{
-    CreateCircuit, CreateCircuitBuilder, SplinterNode, SplinterService,
+    CreateCircuit, CreateCircuitBuilder, SplinterNode, SplinterServiceBuilder,
```

Comment:
Typo in commit messages grXX -> gmXX

Reply:
Should be `gr` (for gameroom). Fixed the code.